packages/codemirror/style/base.css

```diff
@@ -226,3 +226,39 @@
 .cm-s-jupyter .CodeMirror-gutter {
   background-color: var(--jp-layout-color2);
 }
+
+/* Styles for shared cursors (remote cursor locations and selected ranges) */
+.remote-caret {
```

Comment:
We should double check whether this rebase picked up all the changes.

For example there was a quick pass on the CSS in https://github.com/jupyter-rtc/jupyterlab/pull/41/, which resulted in prefixing these lines:

https://github.com/jupyter-rtc/jupyterlab/blob/82c40544f464b455b1c6060e80fcb53a8321355b/packages/codemirror/style/base.css#L231

But here in this diff they don't seem be taken into account.